startDevelopmentStack.sh

```diff
@@ -11,9 +11,9 @@ scp ubuntu@dataland-letsencrypt.duckdns.org:/etc/letsencrypt/live/dataland-local
 ./gradlew dataland-frontend:generateAPIClientFrontend --no-daemon --stacktrace
 
 #start containers for skyminder and edc-dummyserver
-docker-compose --profile development down
-docker-compose --profile development pull
-docker-compose --profile development up -d --build
+docker compose --profile development down
+docker compose --profile development pull
+docker compose --profile development up -d --build
```

Comment:
Why?

Reply:
There were issues on the new server with forwarding docker-compose to docker compose. And on dev1 it was actually forwarded to exactly that.

Reply:
Habe ich nicht ganz verstanden. Können wir vllt später besprechen, wenn ich durch bin mit dem Review.